Doppler.HtmlEditorApi/Controllers/CampaignsController.cs

```diff
@@ -95,7 +95,7 @@ public async Task<IActionResult> SaveCampaign(string accountName, int campaignId
                 });
             }
 
-            if (campaignState.TestABCondition != TestABCondition.TypeClassic)
+            if (campaignState.TestABCondition is not TestABCondition.TypeClassic or TestABCondition.TypeTestABSubject)
```

Comment:
Como funciona esto?

```
            if (campaignState.TestABCondition is (not TestABCondition.TypeClassic) or TestABCondition.TypeTestABSubject)
```

O


```
            if (campaignState.TestABCondition is not (TestABCondition.TypeClassic or TestABCondition.TypeTestABSubject))
```

Yo opino que deberíamos usar simples comparaciones en lugar de pattern matching

```suggestion
            if (campaignState.TestABCondition != TestABCondition.TypeClassic ||  campaignState.TestABCondition != TestABCondition.TypeTestABSubject)
```

O

```suggestion
            if (campaignState.TestABCondition == TestABCondition.TypeTestABContent)
```

Reply:
Cuidado que el mensaje del error debe modificarse

Reply:
Good suggestion. Fixed!